setup.py

```diff
@@ -1,12 +1,13 @@
+# coding=utf-8
 import setuptools
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
 setuptools.setup(
     name="tuyaha",
-    version="0.0.5",
-    author="Pavlo Annekov and original Tuya authors",
+    version="0.0.6",
+    author="ThePinkOwl (Roger Miret Giné), Pavlo Annekov and original Tuya authors",
```

Comment:
Not a good idea, imagine what a long list of contributors will be there. I think the best place is integrated in github itself, check the screenshot, you will be there with us :wink: .
![image](https://user-images.githubusercontent.com/1203892/73138233-1b2e8a00-4069-11ea-910d-a43b4b0b8f5b.png)